Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 1085, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 551, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 846, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 748, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
%matplotlib inline
import os
import sys
import glob
import time

import numpy as np
from astropy.io import fits
import matplotlib.pyplot as plt
from astropy import units as u
import zipfile
import urllib.request

from jwst.pipeline import Detector1Pipeline, Image2Pipeline

from nrm_analysis.misctools import utils
import nrm_analysis
from nrm_analysis.fringefitting.LG_Model import NRM_Model
from nrm_analysis import nrm_core, InstrumentData
from nrm_analysis import find_affine2d_parameters as FAP
from nrm_analysis.misctools.implane2oifits import calibrate_oifits

from nrm_analysis.misctools.utils import Affine2d
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [0;32mIn [1][0m, in [0;36m<module>[0;34m[0m
[1;32m     11[0m [38;5;28;01mimport[39;00m [38;5;21;01mzipfile[39;00m
[1;32m     12[0m [38;5;28;01mimport[39;00m [38;5;21;01murllib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mrequest[39;00m
[0;32m---> 14[0m [38;5;28;01mfrom[39;00m [38;5;21;01mjwst[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpipeline[39;00m [38;5;28;01mimport[39;00m Detector1Pipeline, Image2Pipeline
[1;32m     16[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnrm_analysis[39;00m[38;5;21;01m.[39;00m[38;5;21;01mmisctools[39;00m [38;5;28;01mimport[39;00m utils
[1;32m     17[0m [38;5;28;01mimport[39;00m [38;5;21;01mnrm_analysis[39;00m

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jwst/pipeline/__init__.py:7[0m, in [0;36m<module>[0;34m[0m
[1;32m      5[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01mcalwebb_guider[39;00m [38;5;28;01mimport[39;00m GuiderPipeline
[1;32m      6[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01mcalwebb_image2[39;00m [38;5;28;01mimport[39;00m Image2Pipeline
[0;32m----> 7[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01mcalwebb_image3[39;00m [38;5;28;01mimport[39;00m Image3Pipeline
[1;32m      8[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01mcalwebb_spec2[39;00m [38;5;28;01mimport[39;00m Spec2Pipeline
[1;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01mcalwebb_spec3[39;00m [38;5;28;01mimport[39;00m Spec3Pipeline

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jwst/pipeline/calwebb_image3.py:10[0m, in [0;36m<module>[0;34m[0m
[1;32m      8[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01m.[39;00m[38;5;21;01mresample[39;00m [38;5;28;01mimport[39;00m resample_step
[1;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01m.[39;00m[38;5;21;01moutlier_detection[39;00m [38;5;28;01mimport[39;00m outlier_detection_step
[0;32m---> 10[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01m.[39;00m[38;5;21;01msource_catalog[39;00m [38;5;28;01mimport[39;00m source_catalog_step
[1;32m     12[0m __all__ [38;5;241m=[39m [[38;5;124m'[39m[38;5;124mImage3Pipeline[39m[38;5;124m'[39m]
[1;32m     15[0m [38;5;28;01mclass[39;00m [38;5;21;01mImage3Pipeline[39;00m(Pipeline):

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jwst/source_catalog/__init__.py:1[0m, in [0;36m<module>[0;34m[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01msource_catalog_step[39;00m [38;5;28;01mimport[39;00m SourceCatalogStep
[1;32m      3[0m __all__ [38;5;241m=[39m [[38;5;124m'[39m[38;5;124mSourceCatalogStep[39m[38;5;124m'[39m]

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jwst/source_catalog/source_catalog_step.py:11[0m, in [0;36m<module>[0;34m[0m
[1;32m      8[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcrds[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcore[39;00m[38;5;21;01m.[39;00m[38;5;21;01mexceptions[39;00m [38;5;28;01mimport[39;00m CrdsLookupError
[1;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01mphotutils[39;00m[38;5;21;01m.[39;00m[38;5;21;01mutils[39;00m[38;5;21;01m.[39;00m[38;5;21;01mexceptions[39;00m [38;5;28;01mimport[39;00m NoDetectionsWarning
[0;32m---> 11[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01msource_catalog[39;00m [38;5;28;01mimport[39;00m (ReferenceData, Background, make_kernel,
[1;32m     12[0m                              make_segment_img, calc_total_error,
[1;32m     13[0m                              SourceCatalog)
[1;32m     14[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01m.[39;00m [38;5;28;01mimport[39;00m datamodels
[1;32m     15[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01m.[39;00m[38;5;21;01mstpipe[39;00m [38;5;28;01mimport[39;00m Step

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jwst/source_catalog/source_catalog.py:25[0m, in [0;36m<module>[0;34m[0m
[1;32m     23[0m [38;5;28;01mfrom[39;00m [38;5;21;01mphotutils[39;00m [38;5;28;01mimport[39;00m detect_sources, deblend_sources, source_properties
[1;32m     24[0m [38;5;28;01mfrom[39;00m [38;5;21;01mphotutils[39;00m [38;5;28;01mimport[39;00m CircularAperture, CircularAnnulus, aperture_photometry
[0;32m---> 25[0m [38;5;28;01mfrom[39;00m [38;5;21;01mphotutils[39;00m[38;5;21;01m.[39;00m[38;5;21;01mdetection[39;00m[38;5;21;01m.[39;00m[38;5;21;01mfindstars[39;00m [38;5;28;01mimport[39;00m _StarFinderKernel
[1;32m     26[0m [38;5;28;01mfrom[39;00m [38;5;21;01mphotutils[39;00m[38;5;21;01m.[39;00m[38;5;21;01mutils[39;00m[38;5;21;01m.[39;00m[38;5;21;01m_wcs_helpers[39;00m [38;5;28;01mimport[39;00m _pixel_scale_angle_at_skycoord
[1;32m     28[0m [38;5;28;01mfrom[39;00m [38;5;21;01mjwst[39;00m [38;5;28;01mimport[39;00m __version__ [38;5;28;01mas[39;00m jwst_version

[0;31mModuleNotFoundError[0m: No module named 'photutils.detection.findstars'
ModuleNotFoundError: No module named 'photutils.detection.findstars'

